check/.gitignore

```diff
@@ -0,0 +1 @@
+typediff
```

Comment:
Will you ever have this file on your local machine?  I just `cabal install` `typediff` once and then have it in my path.